src/services/bug/index.ts

```diff
@@ -0,0 +1,24 @@
+import fastifyCors from '@fastify/cors';
+import { FastifyPluginAsync } from 'fastify';
+
+import { BUGS_REPORTING_PREFIX } from '../../utils/config';
+import bugController from './controller';
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  fastify.register(
+    async function (fastify) {
+      // add CORS support
+      if (fastify.corsPluginOptions) {
+        fastify.register(fastifyCors, fastify.corsPluginOptions);
+      }
+
+      // core routes - require authentication
+      fastify.register(async function (fastify) {
+        fastify.register(bugController);
+      });
+    },
+    { prefix: BUGS_REPORTING_PREFIX },
```

Comment:
I think it's clearer if the prefix is clearly stated? Maybe also let's use `report` instead.